api/app/Observers/PoolCandidateObserver.php

```diff
@@ -21,6 +25,30 @@ public function updated(PoolCandidate $poolCandidate): void
     {
         $oldStatus = $poolCandidate->getOriginal('pool_candidate_status');
         $newStatus = $poolCandidate->pool_candidate_status;
+        if (config('feature.notifications')) {
+            if (
```

Comment:
I think this should also check that $newStatus !== $oldStatus

Reply:
Oh yeah!  Good catch.
[only trigger when status changes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10372/commits/888403d2e94d5140a3837eb240b3f931176cf661)